sgqlc/types/__init__.py

```diff
@@ -2016,6 +2018,21 @@ def type(self):
         self._type = self._type.resolve(self.schema)
         return self._type
 
+    @staticmethod
+    def _to_python_name(graphql_name):
+        '''Converts a GraphQL name, ``aName`` to Python: ``a_name``.
+
+        Note that an underscore is appended if the name is a Python keyword.
+        '''
```

Comment:
the CI is pointing out this needs testing, `bin/` is not being checked, but this folder is. The test should be pretty simple, just add doctests here, something like:

```py
>>> BaseItem. _to_python_name('aName')
a_name
>>> BaseItem. _to_python_name('for')
for_
```

Reply:
```suggestion
        >>> BaseItem. _to_python_name('aName')
        a_name
        >>> BaseItem. _to_python_name('for')
        for_
        '''
```